repro/mock_vlm.py

```
@@ -0,0 +1,93 @@
+
+        data = _completion(content)
+        self.send_response(200)
+        self.send_header("Content-Type", "application/json")
+        self.send_header("Content-Length", str(len(data)))
+        self.end_headers()
+        self.wfile.write(data)
+
+
+if __name__ == "__main__":
+    print(f"[mock_vlm] listening on :{PORT} mode={MODE} sleep={SLEEP}", flush=True)
+    ThreadingHTTPServer(("0.0.0.0", PORT), Handler).serve_forever()
```

Ruff (0.15.18)

[error] Possible binding to all interfaces

(S104)